app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import FontAwesome from 'react-fontawesome';
+import React from 'react';
+
+let Comment = React.createClass({
+  propTypes: {
+    /**
+     * An object representing the comment object.
+     */
+    comment: React.PropTypes.shape({
+      /**
+       * Represents how long ago this comment was from. String contains the value and unit of time.
+       */
+      delta: React.PropTypes.string.isRequired,
+
+      /**
+       * The message of the comment.
+       */
+      message: React.PropTypes.string.isRequired,
+
+      /**
+       * The username of the comment writer.
+       */
+      name: React.PropTypes.string.isRequired,
+
+      /**
+       * Represents an array of comments objects which are replies to the current comment.
+       */
+      replies: React.PropTypes.array,
```

Comment:
We can still make this more specific!

```js
React.PropTypes.arrayOf( ... )
```

Always prefer to use `shape` and `arrayOf` rather than the `object` and `array` PropTypes.